app/models/mixins/supports_feature_mixin.rb

```diff
@@ -243,35 +246,48 @@ def feature_known?(feature)
       SupportsFeatureMixin::QUERYABLE_FEATURES.key?(feature.to_sym)
     end
 
+    # These are class variables and it might be modified during runtime because
+    # we dont eager load all classes at boot time, so they needs to be thread
```

Comment:
may i ask you to fix the typo here so i `don't` have to later, please?

Reply:
and `needs`

Reply:
😩

Reply:
Well, I have this fixed locally, but trying to fix tests... and then stumbled on this:

https://github.com/ruby-concurrency/concurrent-ruby/blob/master/lib/concurrent-ruby/concurrent/hash.rb#L17-L21

So that is kinda funny, and kinda makes this comment useless...

Reply:
oh that's cool, alright, well, thanks for being willing to humor me anyway